src/ILVerify/src/Program.cs

```diff
@@ -328,7 +328,10 @@ private void VerifyTypes(PEReader peReader, EcmaModule module, string path, ref
 
         private void PrintVerifyTypesResult(VerificationResult result, EcmaModule module, string pathOrModuleName)
         {
-
+            if (result.Code == VerifierError.InterfaceImplHasDuplicate)
```

Comment:
There is going to be like 100+ different error messages eventually. Doing the specific formatting here for each error message here will make it impossible for somebody (e.g. Roslyn) to take the ILVerifier component (.dll) and get good error messages easily. They will need to duplicate and maintain the formatting logic.

Reply:
ok, do you prefer simple `Console.WriteLine` with already full formatted string? I could try this approach...and if it works in future PRs I could amend current "method IL" report that format directly on driver.

Reply:
I think it would be nice to set it up such that you just call `string Format(string resourceFormat, object[] args)` here to produce the final string.

Reply:
Did some changes...I removed `ErrorArguments` and used only stringformat+args.
Today we need `ErrorArguments` for current IL method error output, but we could move code from driver to ILVerification.dll(it's some work, change resources and centralize report code, but it's possible). What do you think?